controllers/topology/desired_state_test.go

```diff
@@ -19,6 +19,7 @@ package topology
 import (
 	"strings"
 	"testing"
+	"time"
 
 	. "github.com/onsi/gomega"
```

Comment:
This is not straightforward because the Control Plane MachineHealthCheck is generated at the computeDesiredState level directly. It's currently just a check for validity and running computeMachineHealthCheck so I don't think there's any great need to test it here. WDYT?

Reply:
I'll check tomorrow when I re-review the PR